core/src/test/java/io/grpc/CallOptionsTest.java

```diff
@@ -51,17 +51,17 @@
 public class CallOptionsTest {
   private String sampleAuthority = "authority";
   private Long sampleDeadlineNanoTime = 1L;
-  private RequestKey sampleRequestKey = new RequestKey();
+  private Attributes sampleAffinity = Attributes.EMPTY;
```

Comment:
You will want to create a non-empty attributes to differentiate with the default.